### ISSUE TITLE
[Bug]: Recording critical error 11 at src/mesh/RadioLibInterface.cpp:389

### ISSUE THREAD

**Issue by D4rk4** [meshtastic/firmware#2022]
### Category

Other

### Hardware

Rak4631

### Firmware Version

2.0.6

### Description

Same conditions like in https://github.com/meshtastic/firmware/issues/2021

How to repeat: Node connected to a power source (solar,USB), batteries not fully charged (90-95%). Chatty LORA.

HW setup: RAK4631 on RAK19007|RAK19003, RAK1906, w/wo GPS module

### Relevant log output

```Shell
10:15:25 1202 [RadioIf] (bw=250, sf=11, cr=4/8) packet symLen=8 ms, payloadSize=57, time 1042 ms
10:15:25 1202 [RadioIf] Lora RX (id=0x0fc1b46a fr=0xd4 to=0xff, WantAck=0, HopLim=3 Ch=0x8 encrypted rxSNR=6.5 rxRSSI=1.53492e-154)
10:15:25 1202 [RadioIf] AirTime - Packet received : 1042ms
10:15:25 1202 [Router] Found existing packet record for fr=0x25053ad4,to=0xffffffff,id=0xfc1b46a
10:15:25 1202 [Router] Resending implicit ack for a repeated floodmsg
10:15:25 1202 [Router] enqueuing for send (id=0x0fc1b46a fr=0xd4 to=0xff, WantAck=0, HopLim=2 Ch=0x8 encrypted rxSNR=6.5 rxRSSI=1.53492e-154)
10:15:25 1202 [Router] txGood=178,rxGood=359,rxBad=8
10:15:25 1202 [Router] rx_snr found. hop_limit:2 rx_snr:6.500000
10:15:25 1202 [Router] rx_snr found in packet. Setting tx delay:1428
10:15:25 1202 [Router] Found existing packet record for fr=0x25053ad4,to=0xffffffff,id=0xfc1b46a
10:15:25 1202 [Router] Add packet record (id=0x0fc1b46a fr=0xd4 to=0xff, WantAck=0, HopLim=3 Ch=0x8 encrypted rxSNR=6.5 rxRSSI=1.53492e-154)
10:15:25 1202 [Router] Ignoring incoming msg, because we've already seen it (id=0x0fc1b46a fr=0xd4 to=0xff, WantAck=0, HopLim=3 Ch=0x8 encrypted rxSNR=6.5 rxRSSI=1.53492e-154)
10:15:25 1202 [Router] Incoming message was filtered 0x25053ad4
10:15:26 1202 [RadioIf] Starting low level send (id=0x0fc1b46a fr=0xd4 to=0xff, WantAck=0, HopLim=2 Ch=0x8 encrypted rxSNR=6.5 rxRSSI=1.53492e-154 priority=64)
10:15:26 1202 [RadioIf] NOTE! Recording critical error 11 at src/mesh/RadioLibInterface.cpp:389
10:15:26 1202 [RadioIf] Completed sending (id=0x0fc1b46a fr=0xd4 to=0xff, WantAck=0, HopLim=2 Ch=0x8 encrypted rxSNR=6.5 rxRSSI=1.53492e-154 priority=64)
10:15:26 1202 [RadioIf] SX126x standby failed with error -706
WARNING file:stream_interface.py __reader line:179 Meshtastic serial port disconnected, disconnecting... device reports readiness to read but returned no data (device disconnected or multiple access on port?)
```

**Comment by D4rk4** [comment]
Missed config:
`Preferences: { "device": { "serialEnabled": true }, "position": { "positionBroadcastSecs": 120, "positionBroadcastSmartEnabled": true, "gpsEnabled": true, "gpsUpdateInterval": 120, "gpsAttemptTime": 3600, "positionFlags": 3 }, "power": { "onBatteryShutdownAfterSecs": 4294967295, "waitBluetoothSecs": 60, "meshSdsTimeoutSecs": 4294967295, "sdsSecs": 4294967295, "lsSecs": 300, "minWakeSecs": 3600 }, "network": { "ntpServer": "0.pool.ntp.org" }, "display": { "screenOnSecs": 4294967295 }, "lora": { "usePreset": true, "region": "EU_868", "hopLimit": 5, "txEnabled": true, "txPower": 27 }, "bluetooth": { "mode": "FIXED_PIN", "fixedPin": 123456 } }`

**Comment by caveman99** [comment]
Can you please run the firmware with the added debug output, so we know the internal error code?

**Comment by caveman99** [comment]
The PR is not done building yet, sorry... :-) https://github.com/meshtastic/firmware/actions/runs/3619302203 will have a binary (uf2)

**Comment by D4rk4** [comment]
Just flashed 2.0.7.183ec21, collecting logs...

**Comment by D4rk4** [comment]
@caveman99 your build is not working. Just silent reboot. Please check Discord for details.

**Comment by caveman99** [comment]
of course this is not closed - that's what you get for merging debug stuff :-)

**Comment by garthvh** [comment]
Try this build https://github.com/meshtastic/firmware/actions/runs/3677710959

**Comment by D4rk4** [comment]
@garthvh testing...

**Comment by D4rk4** [comment]
@garthvh @thebentern I guess EVENT_LOW_BATTERY event also must be disabled for the nRF platform because it triggers stateSDS. When it's triggered and the battery is charged, the node never turns on back.

**Comment by D4rk4** [comment]
Now much better:
`21:47:11 5616 [RadioIf] (bw=250, sf=11, cr=4/8) packet symLen=8 ms, payloadSize=30, time 722 ms
21:47:11 5616 [RadioIf] Lora RX (id=0x7e1a3db0 fr=0x74 to=0xff, WantAck=0, HopLim=2 Ch=0x8 encrypted rxSNR=6.25 rxRSSI=1.53492e-154)
21:47:11 5616 [RadioIf] AirTime - Packet received : 722ms
21:47:11 5616 [RadioIf] Starting low level send (id=0x6d1bf14f fr=0x6c to=0xff, WantAck=0, HopLim=5 Ch=0x8 encrypted rxtime=1670881630 priority=1)
21:47:11 5616 [RadioIf] SX126x standby failed with error -706`

After that device rebooted by watchdog

**Comment by garthvh** [comment]
There is no super deep sleep for NRF anymore.

**Comment by garthvh** [comment]
Fixed in 2.0.7

**Comment by D4rk4** [comment]
@garthvh @thebentern partially fixed in 2.0.7, node still killed by EVENT_LOW_BATTERY, when bat around 67%

https://discord.com/channels/867578229534359593/919642584480112750/1052188924073746432

`'deviceMetrics': {'batteryLevel': 67, 'voltage': 3.896, 'channelUtilization': 30.26, 'airUtilTx': 1.3934444}}`
